TrinityCore/PlayerMethods.h

```diff
@@ -3393,9 +3393,9 @@ namespace LuaPlayer
     /**
      * Adds a new item to the gossip menu shown to the [Player] on next call to [Player:GossipSendMenu].
      *
-     * sender and intid are numbers which are passed directly to the gossip selection handler. Internally they are partly used for the database gossip handling.  
-     * code specifies whether to show a box to insert text to. The player inserted text is passed to the gossip selection handler.  
-     * money specifies an amount of money the player needs to have to click the option. An error message is shown if the player doesn't have enough money.  
```

Comment:
These whitespaces are new lines because this is markdown.
Maybe they could be replaced with <br /> instead of just removing.

Reply:
those got auto removed on save, ill revert that

Reply:
done